__tests__/nc-games.test.js

```diff
@@ -271,6 +271,59 @@ describe("/api/reviews/:review_id/comments", () => {
             })
         })
     })
+
+    describe("POST", () => {
+        test("Status 201: When POST is used on the ./comments endpoint and passed an object with a username and body, returns the posted comment.", () => {
+            return request(app)
+            .post("/api/reviews/12/comments")
+            .send({username: 'bainesface', body: 'Scythe is Lythe'})
+            .expect(201)
+            .then(({body}) => {
+                const { postedComment } = body;
```

Comment:
The keys we send information back on should be pretty consistent and uniform where possible. I'd like this POST response to come back on the key of `comment` instead of `postedComment` if possible